incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,58 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
+  tag: "latest"
```

Comment:
Immutable references are always better, but `3` would be ok. We do expect the images in the charts to be regularly updated.